examples/misc_controls_transform.html

```diff
@@ -89,7 +88,9 @@
 				scene.add( mesh );
 
 				control.attach( mesh );
-				scene.add( control );
+
+				const gizmo = control.getGizmo();
```

Comment:
@mrdoob Are you okay with `getGizmo()`?

Reply:
@arodic How does this change look to you?

Reply:
`getGizmo()` is okay 👌

Reply:
Actually, `getHelper()` would follow the current naming: 
https://github.com/mrdoob/three.js/tree/dev/examples/jsm/helpers